packages/old-import/src/widgets/definitions/rss.ts

```diff
@@ -4,6 +4,7 @@ export type OldmarrRssDefinition = CommonOldmarrWidgetDefinition<
   "rss",
   {
     rssFeedUrl: string[];
+    anableRtl: boolean;
```

Comment:
Typo, would be `enableRtl`

Okay I first thought why we would need an option for that as we already could use `useDirection`, but it makes sense as the source of the RSS feed has obviously not to be in the same language as the user choose.